src/js/components/Tiles.js

```diff
@@ -195,14 +207,23 @@ export default class IndexTiles extends Component {
       }, this);
     }
 
+    let Actions;
+
+    if (actionsComponent) {
+      Actions = actionsComponent;
+    }
+
     return (
-      <Tiles className={classes.join(' ')} onMore={onMore}
-        flush={this.props.flush} fill={this.props.fill}
-        selectable={this.props.onSelect ? true : false}
-        selected={selectionIndex}
-        size={this.props.size}>
-        {tiles}
-      </Tiles>
+      <Box direction="row" responsive={false}>
+        <Tiles className={classes.join(' ')} onMore={onMore}
+          flush={this.props.flush} fill={this.props.fill}
+          selectable={this.props.onSelect ? true : false}
+          selected={selectionIndex}
+          size={this.props.size}>
+          {tiles}
+        </Tiles>
+        {Actions && <Actions items={result.items}/>}
```

Comment:
Instead of putting this to the right of the Tiles. I think we should always have a section header, even when there aren't separate sections defined. Tracy, Chris, and I also discussed moving the two numbers above and below the filter control icon into this section header. That would make it clearer what the bulk actions correspond to.